packages/fluentui/react-northstar/src/components/Form/FormInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { compose, ComponentWithAs, ShorthandConfig } from '@fluentui/react-bindings';
```

Comment:
what is the difference betwen form field and FormInput? Should we add a note in the docs explaining when should which one be used?

Also, it would be good to add/remove examples